### PR TITLE
feat: Rewrote Managed Service section for SaaS & Enterprise.

### PR DESCRIPTION
**Pull request by Jelledb**

### CODE REVIEW

File: docs/meshstack.managed-service.md [hunk 1]
```diff
@@ -3,78 +3,157 @@ id: meshstack.managed-service
...
+### SaaS: Infrastructure
 
-An on-premise CI/CD pipeline automatically rolls out new releases to a staging environment. This allows customers to test
-new functionality, bug fixes or configuration changes.
+Each meshStack SaaS installation is managed in a standardized way, and it is not possible to deviate from this
+standard unless the installation is upgraded to the [meshStack Enterprise plan](#meshstack-enterprise).
 
-### Production Environment
+meshStack SaaS is hosted on a Google Cloud-managed kubernetes cluster (this service is known as GKE).
+Each individual meshStack SaaS environment is hosted in its own so-called Kubernetes namespace.
```
Comment (malhussan) [inline docs/meshstack.managed-service.md:93]:
```suggestion
Each individual meshStack SaaS environment is hosted in its own so-called Kubernetes namespace secured with a network policy blocking traffic from other meshStacks.
```

File: docs/meshstack.managed-service.md [hunk 2]
```diff
@@ -3,78 +3,157 @@ id: meshstack.managed-service
...
 
-meshcloud maintains meshStack as a rolling release. This means that releases follow a "fix-forward" paradigm and
-don't support (nor require) rollbacks. Any release is generally considered "End of Life" (EOL) as soon as a release
-with a higher version number becomes available. meshcloud's managed service supports operation for EOL releases for
-a limited time period (upgrade window) before the customer is expected to approve an update to the next release.
+The infrastructure of a meshStack Enterprise installation always depends on what is agreed upon with the customer.
 
-meshcloud can offer extended support for feature releases that have officially reached end of life as part of an individual managed service agreement. Extended support covers managed operation of an EOL release as well as a
-commitment to providing further patch releases for critical bugs and security issues found in a given feature release.
+meshcloud supports both on-premise installations but also public cloud (AWS, Azure & Google Cloud) installations
```
Comment (malhussan) [inline docs/meshstack.managed-service.md:119]:
```suggestion
meshcloud supports both on-premise installations and public cloud (AWS, Azure & Google Cloud) installations
```

File: docs/meshstack.managed-service.md [hunk 3]
```diff
@@ -3,78 +3,157 @@ id: meshstack.managed-service
...
+
+#### Rolling out Staging Environment
+
+A CI/CD pipeline (either one from meshcloud, or your own) automatically rolls out new releases to the staging / QA environment.
+This allows you to test new releases as soon as possible. The timing of rollouts can be decided as desired,
+e.g. as soon as possible or at a certain time window.
+
+#### Rolling out Production Environment
+
+Production releases are also automated via a CI/CD pipeline but are typically not automatically triggered by meshcloud.
```
Comment (malhussan) [inline docs/meshstack.managed-service.md:153]:
```suggestion
Production releases are also automated via a CI/CD pipeline but rollouts are typically not automatically triggered by meshcloud.
```

File: docs/meshstack.managed-service.md [hunk 4]
```diff
@@ -3,78 +3,157 @@ id: meshstack.managed-service
...
+The production environment of meshStack Enterprise is the real environment where all your users will be consuming
+your cloud foundation services.
+The production environment is managed by meshcloud.
+The timeline of rolling out new releases can be decided by the customer (more on that below).
+
+### Enterprise: Rollouts
+
+#### Rolling out Staging Environment
+
+A CI/CD pipeline (either one from meshcloud, or your own) automatically rolls out new releases to the staging / QA environment.
```
Comment (henryde) [inline docs/meshstack.managed-service.md:147]: Pipelines are always managed by us they may just run on a customer provided CI/CD system.

**Review by henryde — APPROVED** [review]

**Pull request merged**